lib/inflections/en.php

```diff
@@ -39,6 +39,9 @@
 	->plural('/^(ox)$/i', '\1en')
 	->plural('/^(oxen)$/i', '\1')
 	->plural('/(quiz)$/i', '\1zes')
+	->plural('/^is$/', 'are')
```

Comment:
`is`, `was`, `has` are verbs, not nouns. Please remove them.

Reply:
@olvlvl I realise they are verbs, but they are quite useful when showing notifications in a website and you have a verb based on a subject which could be a number of items of 0, 1 or more than 1.

> 1 product was added to your cart

> 2 products were added to your cart

I'd use an inflector here on both `product`/`products` and `was`/`were`.